tox.ini

```diff
@@ -28,14 +28,18 @@ extras =
     lxc
     openstack
     vagrant
+commands_pre =
+    find {toxinidir} -type f -not -path '{toxinidir}/.tox/*' -not -path '*/__pycache__/*' -name '*.py[c|o]' -delete
 commands =
     unit: pytest test/unit/ --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
     functional: pytest test/functional/ {posargs}
     lint: flake8
     lint: yamllint -s test/ molecule/
+whitelist_externals =
+    find
 
 [testenv:lint]
-deps = 
+deps =
```

Comment:
Please try to separate logical changes in the future. Even this small...

Reply:
Sure. I will try. Even better, I do know how to avoid the need for such fixes with some additional pre-commit hooks.

Reply:
Well, yeah... `-n` or/and `-p` are your friends in Git :)